src/components/about/karusellProjektbeteiligte.svelte

```diff
@@ -0,0 +1,254 @@
+			</button>
+			<!-- Karte bestehend aus tabelle und bild -->
+			<div
+				bind:this={elemCarousel}
+				class="snap-x snap-mandatory scroll-smooth flex overflow-x-auto"
+			>
+				{#each tableSources as source}
+					<div class="snap-center rounded-container-token carousel-item">
+						<!-- Bild-Teil der Karte -->
+						<div class="image-container">
+							<img class="carousel-image" src={source.src} alt={source.id} loading="lazy" />
+						</div>
+						<!-- Text-Teil der Karte -->
+						<div class="table-container">
+							<Table source={source.table} />
+						</div>
+					</div>
+					<span class="divider-vertical h-72" />
+				{/each}
+			</div>
+			<!-- Button: Right -->
+			<button type="button" class="btn-icon variant-filled" on:click={carouselRight}>
+				<img src="./src/images/icons/arrow-right.png" alt="DHBW Mannheim Logo" />
+			</button>
+		</div>
+	</svelte:fragment>
+</AccordionItem>
+
+<style>
+	/* Definition von CSS-Variablen für wiederkehrende Werte */
+	:root {
+		--carousel-item-min-width: 600px; /* Mindestbreite für Karussell-Elemente */
+		--carousel-item-margin: 20px; /* Außenabstand für Karussell-Elemente */
+		--table-container-min-width: 400px; /* Mindestbreite für Tabellencontainer */
+		--table-container-min-height: 300px; /* Mindesthöhe für Tabellencontainer */
+		--image-container-min-width: 200px; /* Mindestbreite für Bildcontainer */
+		--image-container-min-height: 200px; /* Mindesthöhe für Bildcontainer */
+		--carousel-image-max-width: 200px; /* Maximale Breite für Karussell-Bilder */
+		--carousel-image-max-height: 200px; /* Maximale Höhe für Karussell-Bilder */
+		--logo-icon-size: 35px; /* Größe für Logo-Icons */
+	}
+
+	/* Stile für Karussell-Elemente */
+	.carousel-item {
+		display: flex;
+		flex-direction: row;
+		width: 100%;
+		justify-content: center;
+		min-width: var(--carousel-item-min-width);
+		margin-right: var(--carousel-item-margin);
+		margin-left: var(--carousel-item-margin);
+	}
+
+	/* Stile für Tabellencontainer */
+	.table-container {
+		text-align: left;
+		width: 100%;
+		word-wrap: break-word;
+		min-width: var(--table-container-min-width);
+		min-height: var(--table-container-min-height);
+	}
+
+	/* Stile für Bildcontainer */
+	.image-container {
+		width: 100%;
+		display: flex;
+		justify-content: center;
+		min-width: var(--image-container-min-width);
+		min-height: var(--image-container-min-height);
+	}
+
+	/* Stile für Bilder im Karussell */
+	.carousel-image {
+		border-radius: 50%;
+		overflow: hidden;
+		width: 100%;
+		height: auto;
+		object-fit: cover; /* Sorgt dafür, dass das Bild den Container ausfüllt */
+		max-width: var(--carousel-image-max-width);
+		max-height: var(--carousel-image-max-height);
+	}
+
+	/* Stile für verschiedene Icons */
+	.icon-team {
+		height: var(--logo-icon-size);
+		width: var(--logo-icon-size);
+	}
+
+	/* Media Query für kleine Bildschirme (z.B. Mobiltelefone) */
+	@media (max-width: 600px) {
```

Comment:
Eventuell anpassen tailwind css: https://tailwindcss.com/docs/responsive-design

Reply:
In dem Zusammenhang kann man sich auch überlegen, ob man anstatt `px `nicht lieber `rem `benutzt

Reply:
Ich habe jetzt entsprechend der doku 640px verwendet wurde:, siehe link von Paul

Reply:
habe jetzt doch rem verwendet